ModList/UI/Components/TextMeshProUGUILinkBackgroundGenerator.cs

```diff
@@ -22,24 +21,26 @@ public class TextMeshProUGUILinkBackgroundGenerator : MonoBehaviour
         public Transform BackgroundParent { get; set; }
 
         private bool useLineHeight = false;
+
         public bool UseLineHeight
         {
             get => useLineHeight;
             set
             {
                 useLineHeight = value;
-                needsRerender = true;
+                _needsRerender = true;
```

Comment:
I also mentioned this in DMs; I really dislike the `_` prefix for private fields.